core/src/main/scala/kafka/log/Log.scala

```diff
@@ -274,20 +274,24 @@ class Log(@volatile var dir: File,
       val filename = file.getName
       if (filename.endsWith(DeletedFileSuffix) || filename.endsWith(CleanedFileSuffix)) {
         // if the file ends in .deleted or .cleaned, delete it
-        Files.deleteIfExists(file.toPath)
+        file.delete()
       } else if(filename.endsWith(SwapFileSuffix)) {
         // we crashed in the middle of a swap operation, to recover:
         // if a log, delete the .index file, complete the swap operation later
         // if an index just delete it, it will be rebuilt
         val baseFile = new File(CoreUtils.replaceSuffix(file.getPath, SwapFileSuffix, ""))
         if (isIndexFile(baseFile)) {
-          Files.deleteIfExists(file.toPath)
+          if (!file.delete())
+            info(s"Index file ${file.getAbsolutePath} not deleted")
```

Comment:
Perhaps "Index file ... **was** not deleted" or "Deleting the index file ... was not successful" (just to make it a complete sentence). This also applies to other added log messages.